requirements.txt

```diff
@@ -0,0 +1,10 @@
+pip>=9
+setuptools>=26
+wheel>=0.29
```

Comment:
Are those really required? They are always expected to be up to date but comes as base python setup

Reply:
`pip` and `setuptools` do but I am not sure for `wheel`.

Typically, the logs of CI checks show that `wheel` was downloaded and installed based on the requirements (requirement was not already satisfied). 

I am removing the two first ones, what do you think about `wheel`? Would you still remove it?

Reply:
No need to bother, we can check later on, I was mostly curious :sweat_smile: